registry/session.go

```diff
@@ -308,9 +315,10 @@ func (r *Session) GetRemoteImageLayer(imgID, registry string, imgSize int64) (io
 	return res.Body, nil
 }
 
+// GetRemoteTag retrieves a tag from the given repository
```

Comment:
`askedTag` refers to the tag as requested by the user. Usually the term asked is used when the tag may also be "empty", indicating a user did not ask for a tag but one was defaulted or multi tags were used. In this case the variable should likely just be named `tag`.